packages/remix-dev/server-build.ts

```diff
@@ -1,11 +1,11 @@
 import type { ServerBuild } from "@remix-run/server-runtime";
 
+export const assets: ServerBuild["assets"] = undefined!;
```

Comment:
Let's revert this. Code changes are merged into `dev` until release, only docs and examples should get merged into `main`. After the release we merge everything back into `main` and `dev` from the release branch. We can live with the warning for now.